static/docs/install/linux.md

```diff
@@ -12,10 +12,9 @@
 $ pip install dvc
 ```
 
-Depending on the type of the
-[remote storage](/doc/user-guide/external-dependencies) you plan to use, you
```

Comment:
`/doc/user-guide/external-dependencies`  is a bug! I'm not sure why do we have this link at all here.

Reply:
I'm fixing this right away ... no reason to wait for fixes like this

Reply:
Fixed by https://github.com/iterative/dvc.org/commit/50bc171793b31b6d94c35baaccf7e348e8e096a7

Reply:
You have fixed it in a way that creates conflicts, not as part of this PR.

Reply:
Yep. Resolving conflicts is a very regular part of the modern dev process. That's why git has rebase, merge, etc, etc. Even Github can resolve automatically majority of conflicts.

We can't wait too long to merge this to fix the existing bug.